frappe/core/doctype/page/page.py

```diff
@@ -104,8 +104,18 @@ def as_dict(self, no_nulls=False):
 		return d
 
 	def on_trash(self):
+		if not frappe.conf.developer_mode:
+			frappe.throw(_("Deletion of this document is only permitted in developer mode."))
```

Comment:
@ankush this makes it kinda hard to migrate since patches.txt contains a couple of `execute:frappe.delete_doc('Page', ...)`

Reply:
@barredterra thanks for pointing out, we can ignore this in migrate.

Reply:
Patch test ig is just passing because there are no new patches between v15 and develop :see_no_evil: